### PR TITLE
Add recipe for fisx 1.1.6

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Comment by tschoonj** [comment]
I would like to add this recipe to conda-forge as it was the last remaining dependency of PyMca that wasn't here yet.

So when merged in, my next PR will be for PyMca :smile:

File: recipes/fisx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Quantitative X-Ray Fluorescence Analysis Support Library
+  description: |
+          This software library implements formulas to calculate, given an experimental setup, the expected x-ray fluorescence intensities.
+          The library accounts for secondary and tertiary excitation, K, L and M shell emission lines and de-excitation cascade effects.
+          The basic implementation is written in C++ and a Python binding is provided.
+
+
```
Comment (marcelotrevisani) [inline recipes/fisx/meta.yaml]: Please, remove this extra empty line here

Comment (tschoonj) [reply recipes/fisx/meta.yaml]: Done!

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge`` !  :)

**Pull request merged**